src/types.ts

```diff
@@ -80,4 +80,28 @@ export namespace StableDiffusionWebUI {
     parameters: any
     info: any
   }
+
+  /**
+   * @see https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/master/modules/api/models.py#L122
```

Comment:
prefer permalink because master code is subject to change

Reply:
OK, will change later.